values.md

```diff
@@ -1,6 +1,6 @@
-### Profits are exhaust.
+### Profits fund our mission
 
-We are a mission-driven company. We believe that profits are an important tool in the pursuit of our mission, but they are not the mission itself.
+We are a mission-driven company. Our software, company, and community have been designed to improve the lives of real people. Commercial success fuels our pursuit of the mission.
```

Comment:
I think each instance makes sense in a vacuum, but changing multiple values to include the phrase "commercial success" has a feel to it. To be super clear, I really like the way it reads in this one in particular.

Reply:
@ian-fahey-dbt this is a great observation. I do think it can be important to let our values stand alone and even say seemingly conflicting things. Resolving the tension between values is the work of, well, existing. 

So letting the profits-related value be the one about commercial success, and then living out these values by putting these values in conversation with each other navigating their tensions - feels like the way to go to me. (i.e., rather than having similar wording in multiple values)

Reply:
> Commercial success fuels our pursuit of the mission

I know its verbose, but this is powerful enough to me to be the value! It's not about profits, but profits are crucial. `fund` is such a mechanical verb, `fuels` is really talking about what we're aiming to do here. We have a great vehicle and we want it to move forward, for the community at large.

`Commercial success` too is great! It implies profits but not at the cost of other factors. It's ultimately what a good business should be pursuing instead of _just_ profits

Reply:
Really appreciating the rich conversation around this topic. Part of what resonated about the original value was the clear sense of prioritizing our mission and direction as our focus. It defines a relationship between mission and profits where profits are one tool (out of many) that fuels our mission. While I'm generally a big proponent of defining the positive vs. negative as effective wayfinding/direction-setting, I don't actually agree that the value as it stands is in the negative. It clearly names a relationship (mission as focus, profits as supporting tool and consequence of mission) that is HUGELY in contrast to so many organizations (that many of us have been shaped by).

I will also name that this period in the company's growth has had a lot of focus on revenue, improving margins, etc. and it's uncomfortable to hold those conversations alongside the value "profits are exhaust." What an exciting opportunity to live into this tension together! ;) I would propose holding off on changing this value during this particular season and rather live into the collective discomfort a bit longer. I believe there's something deeper/clearer waiting to surface in this conversation.